app/views/components/_glance-metric.html.erb

```diff
@@ -22,31 +22,5 @@
       <% end %>
     </span>
     <p class="app-c-glance-metric__context govuk-body-xs"><%= context %></p>
-    <span class="app-c-glance-metric__trend">
-      <% if trend_percentage.blank? %>
```

Comment:
Could `trend_percentage` be removed too, or is this needed elsewhere?

Reply:
Good point, I'm hoping that this is a temporary change so I've actually tried to keep as much of the code in as possible, I ended up just removing this block though.

Hopefully it can be decided soon whether to properly remove this as a feature, or make it reliably work again!